docs/proposals/20200506-conditions.md

```diff
@@ -243,8 +243,8 @@ When the above suffix are not adequate for a specific condition type, other suff
 (e.g. `Completed`, `Healthy`); however, it is recommended to balance this flexibility with the objective to provide
 a consistent condition naming across all the Cluster API objects.
 
-The `Severity` field MUST be set only when `Status=False` and it is designed to provide a standard classification
-of possible conditions failure `Reason`. 
+The `Severity` field MUST be set only when `Status=False` + positive polarity or when `Status=True` + negative polarity;
+severity it is designed to provide a standard classification of possible conditions failure `Reason`. 
```

Comment:
```suggestion
severity is designed to provide a standard classification of possible conditions failure `Reason`. 
```

Reply:
I agree the wording is a touch confusing, I read this as: 

Severity must only be set when something of positive polarity has 'Status=False' (something good is not true), or when something of negative polarity has 'Status =True'  (something bad is true)